Library/Homebrew/cask/lib/hbc/installer.rb

```diff
@@ -87,7 +87,7 @@ def summary
     s = if MacOS.version >= :lion && !ENV["HOMEBREW_NO_EMOJI"]
           (ENV["HOMEBREW_INSTALL_BADGE"] || "\xf0\x9f\x8d\xba") + "  "
         else
-          "#{Hbc::Utils::Tty.blue.bold}==>#{Hbc::Utils::Tty.reset.bold} Success!#{Hbc::Utils::Tty.reset} "
+          "#{Tty.blue}==> Success!#{Tty.reset} "
```

Comment:
There's `Tty.highlight` (slightly more semantic name), that is currently defined as “bold using the foreground color”, so that would work as a replacement. It's used in `pretty_installed` that is used for highlighting installed formulae in update reports and `brew info`. Somewhat confusingly, there's also `Tty.white` with the exact same definition and that's what used in `oh1` and `ohai` for bolding the text after the arrow.

I suggest to use the latter for consistency with `oh1` and `ohai`.